### ISSUE TITLE
Quiz 2 - Blockchain

### ISSUE THREAD

**Issue by allluz** [w3b3d3v/squad-midiatico-content-PT#142]

**Comment by s4mp41xao** [comment]
Para evitar fraudes, a Blockchain exige um trabalho adicional para inclusão de novos blocos: trata-se:

do livro-razão (ledger)
da prova de trabalho (proof of work)
da marcação de tempo (timestamp)
do nível de dificuldade (difficult target)

Resposta correta: do livro-razão (ledger)

**Comment by s4mp41xao** [comment]
@allluz Verdade. acabei esquecendo. Uma sugestão, sugiro que seja melhor publicar em formato de Poll no LinkedIn, mesmo que seja um quiz. Apesar de não ser possível colocar uma opção correta, acho que seria mais proveitoso do que comentar.
